patch/0025-net-udp_l3mdev_accept-support.patch

```diff
@@ -0,0 +1,279 @@
+From 58be55eb5de3a319c0740ba8071440d972c821c3 Mon Sep 17 00:00:00 2001
+From: Harish Venkatraman <Harish_Venkatraman@dell.com>
+Date: Tue, 25 Sep 2018 09:56:25 -0700
+Subject: [PATCH] net: udp_l3mdev_accept support
```

Comment:
is this upstream patch? can you put upstream commit it in the commit message?

Reply:
This is a upstream patch which was back ported to 4.9 kernel manually since there is no patch available for 4.9.

> We are planning to move to 4.9.0-8 soon for security fixes, would this fix be compatible with that version? Since this change touch the key modules in the kernel, it would to good to upstream it first if not yet done or share the upstream link if already exists.

This is a patch which is back ported to 4.9 for SONiC. Secondly this fix is self contained for UDP and should be compatible.

Reply:
I guess the changes you put here originate from few upstream commits after 4.9 though they may not be a exact match.  Could you you list  those upstream commits in your commit message?

Reply:
@jipanyang I have updated the Original commit message with the changes